core/src/main/java/com/softwaremill/session/javadsl/CsrfDirectives.scala

```diff
@@ -12,7 +12,7 @@ import com.softwaremill.session.CsrfCheckMode
 trait CsrfDirectives {
 
   def randomTokenCsrfProtection[T](checkMode: CsrfCheckMode[T], inner: Supplier[Route]): Route = RouteAdapter {
```

Comment:
we probably should have both `randomTokenCsrfProtection` (for backwards compatibility) and `hmacTokenCsrfProtection` method in the java api as well?

Reply:
Done.